satpy/readers/ami_l1b.py

```diff
@@ -110,9 +91,9 @@ def get_area_def(self, dsid):
         bit_shift = 2**16
         area_extent = (
             h * np.deg2rad((0 - coff - 0.5) * bit_shift / cfac),
-            h * np.deg2rad((0 - loff - 0.5) * bit_shift / lfac),
+            -h * np.deg2rad((0 - loff - 0.5) * bit_shift / lfac),
             h * np.deg2rad((cols - coff + 0.5) * bit_shift / cfac),
-            h * np.deg2rad((rows - loff + 0.5) * bit_shift / lfac),
+            -h * np.deg2rad((rows - loff + 0.5) * bit_shift / lfac),
```

Comment:
Can you rebase on my branch since it already includes this change.

Reply:
I think I've done this...